src/components/templates/Reports.svelte

```diff
@@ -71,6 +81,6 @@
   </section>
 
   <section>
-    <!-- Insert donut charts here -->
+    <DonutSection />
```

Comment:
This is now a section within a section (the donut chart itself also contains a section)

Reply:
can this be solved by using a Div? due to all the components it is very difficult to position the items in a grid without a positionable container like a section or a div